src/typescript/11-keyof-typeof/keyof-typeof.md

```diff
@@ -85,21 +91,19 @@ interface Dinosaur {
 `typeof` is often used in conjunction with `ReturnType`, a utility type (learn more about utility types: [learn-typescript/utility-types]) provided by TypeScript, as a way to type out the return of a function. Say for example we have a function defined somewhere and we need to give a type for a value that is returned from the function. We could achieve this using `ReturnType`
```

Comment:
I think these sentences would read better if we said “the `typeof` operator` instead of just `typeof`.

`Say for example we` -> `Say, for example, we`